### PR TITLE
Fixed misc. RDLI test issues.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Fixed some bugs in the tests that occur when you use an existing instance.
- Fixing bug in the "start-fake" command.

### CODE REVIEW

File: tests/integration/tests/api/instances.py [hunk 1]
```diff
@@ -648,7 +648,8 @@ def test_get_legacy_status_notfound(self):
     def test_volume_found(self):
         instance = dbaas.instances.get(instance_info.id)
         assert_equal(instance_info.volume['size'], instance.volume['size'])
-        assert_true(0.12 < instance.volume['used'] < 0.25)
+        if create_new_instance():
```
Comment (rnirmal) [inline tests/integration/tests/api/instances.py:651]: why on if create_new_instance(). If there's an existing instance, it should return the used volume

File: tests/integration/tests/api/instances.py [hunk 2]
```diff
@@ -874,6 +875,8 @@ def volume(self):
             return
         if self.volume_key_exists():
             expected_attrs = ['size']
+            if not create_new_instance():
```
Comment (rnirmal) [inline tests/integration/tests/api/instances.py:878]: other way round if it's a new instance and it's active, the used volume should be available

**Comment by ed-** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**